packages/editor/src/components/autosave-monitor/index.native.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
+
+/**
+ * External dependencies
+ */
+import RNReactNativeGutenbergBridge from 'react-native-gutenberg-bridge';
+
+export class AutosaveMonitor extends Component {
+	componentDidUpdate( prevProps ) {
```

Comment:
Hmm, it's very strange that we are using Component which basically won't be rendered (as it's null) to hold AutosaveMonitor logic. Is it possible to avoid AutosaveMonitor component to extend Component?

Reply:
I guess it may result in a difference in how the HOC works: https://github.com/WordPress/gutenberg/blob/ce78cd88e7088cb60cb2b9395e756c9452fd9ebf/packages/compose/src/pure/index.js#L24, not :100: sure though, as I didn't deeply examine this.

Reply:
> I guess it may result in a difference in how the HOC works:
> 
> https://github.com/WordPress/gutenberg/blob/ce78cd88e7088cb60cb2b9395e756c9452fd9ebf/packages/compose/src/pure/index.js#L24
> 
> , not 💯 sure though, as I didn't deeply examine this.

Yes, there are different ways of doing this, but I think it's better for us to stay close to the web implementation.